Makefile

```diff
@@ -23,7 +23,13 @@ run-mock-registry: ${GOPATH}/bin/mock-registry vendor
 prepare-build: install
 	cp "${GOPATH}"/bin/broker build/broker
 
-build: prepare-build
+dev-image: prepare-build
+	docker build -f ${BUILD_DIR}/Dockerfile-src ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	@echo
+	@echo "Remember you need to push your image before calling make deploy"
+	@echo "    docker push ${BROKER_IMAGE}:${TAG}"
+
+release-image:
```

Comment:
@rthallisey I suggested in another comment that we use a `make release` that calls `release-image`. For now it will *NOT* do a push. I want to make the push explicit. If it becomes the norm that make release is always followed by a push, then we simply update the target to include push.